imports/plugins/core/orders/server/no-meteor/mutations/cancelOrderItem.js

```diff
@@ -107,6 +111,9 @@ export default async function cancelOrderItem(context, input) {
         quantity: cancelQuantity
       };
 
+      // Update the subtotal since it is related to the quantity
+      updatedItem.subtotal = item.price.amount * cancelQuantity;
```

Comment:
Haven't tested this in action yet, but based on fixing a similar issue when adding items to a cart, I believe that `subtotal` is an object with `amount` and `currencyCode` on it, not just a `number`.

Will confirm / disprove when testing.

Reply:
Yes, this confused me at first, too. Love these little quirks!